src/parser/seclang-scanner.ll

```diff
@@ -658,7 +658,6 @@ EQUALS_MINUS                            (?i:=\-)
 
 <EXPECTING_ACTION_PREDICATE_VARIABLE>{
 [}] { BEGIN_PREVIOUS(); }
-[}][%] { BEGIN_PREVIOUS(); }
```

Comment:
Rigth, we need to be very carfully here, as both representation are valid:

- ```%{ARGS_COMBINED_SIZE}%``` 
- ```%{ARGS_COMBINED_SIZE}```

Considering those cases as valid, we may need to came up with something similar to **(not tested)**:

```
(...)
<EXPECTING_ACTION_PREDICATE_VARIABLE>{
[}][%][{] { yyless(2); BEGIN_PREVIOUS(); }
[}][%]    { BEGIN_PREVIOUS(); }
[}]       { BEGIN_PREVIOUS(); }
}
(...)
```

Once we realize that the subsequent % belongs to a new variable, we return the tokens to lex and 'BEGIN_PREVIOUS' state. Let me know what you think about it.